bpo-3530: Use fix_missing_locations when node transformer adds nodes (#17172)
python/cpython: Misc/NEWS.d/next/Documentation/2019-11-17-11-53-10.bpo-3530.8zFUMc.rst

> In ast doc, fix misleading NodeTransformer example and add

Reviewer: Please, add markup formatting to NodeTransformer and fix_missing_locations

> In ast doc, fix misleading NodeTransformer example and add

Reviewer: Also: in ast doc -> in the :mod:`ast` module documentation.

Reviewer: You didn't apply this suggestion, did you? ("in the :mod:`ast` module documentation...")

> in the :mod:`ast` module documentation, fix a misleading ``NodeTransformer`` example and add

Reviewer: One last typo:

```suggestion
In the :mod:`ast` module documentation, fix a misleading ``NodeTransformer`` example and add
```

> In :mod:`ast` doc, fix misleading ``NodeTransformer`` example and add

Reviewer: ```suggestion
in the :mod:`ast` module documentation, fix a misleading ``NodeTransformer`` example and add
```

> ``fix_missing_locations`` advice.

Reviewer: ```suggestion
advice on when to use the ``fix_missing_locations`` function.
```
